ci/tests/e2e-helm.sh

```
@@ -137,24 +137,47 @@ EOF
+# available. Split out from the main flow so CI jobs that only need a running
+# operator (e.g. a TLS scanner step) can invoke deployment via DEPLOY_ONLY
+# without also running test_operator/undeploy below.
+deploy_operator() {
+    # Give the serviceaccount cluster role to create statefulsets
+    if oc api-versions | grep openshift; then
+        oc adm policy add-cluster-role-to-user cluster-admin -z default || :
+    fi
+
+    # deploy operator
+    echo "running make deploy"
+    make deploy IMG=$IMAGE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)